os/net/mac/tsch/tsch.c

```diff
@@ -847,7 +848,7 @@ PROCESS_THREAD(tsch_send_eb_process, ev, data)
 
   /* Set an initial delay except for coordinator, which should send an EB asap */
   if(!tsch_is_coordinator) {
-    etimer_set(&eb_timer, random_rand() % TSCH_EB_PERIOD);
+    etimer_set(&eb_timer, TSCH_EB_PERIOD ? random_rand() % TSCH_EB_PERIOD : 0);
```

Comment:
Out of pure curiosity, what is the purpose of `TSCH_EB_PERIOD == 0`?
Looks like we might end up in an infinite busy-wait with this setting here, no?